redisvl/redis/utils.py

```diff
@@ -40,6 +40,9 @@ def buffer_to_array(buffer: bytes, dtype: Any = np.float32) -> List[float]:
     return np.frombuffer(buffer, dtype=dtype).tolist()
 
 
-def hashify(content: str) -> str:
-    """Create a secure hash of some arbitrary input text."""
+def hashify(content: str, extras: Optional[Dict[str, Any]] = None) -> str:
+    """Create a secure hash of some arbitrary input text and optional dictionary."""
+    if extras:
```

Comment:
Small preference here to keep `hashify` dumb and move the filter string processing logic for this to within the `generate_id` method and pass the final string to hashify.